### DISCUSSION TITLE
Single Node Assisted Installer required node network to be at last /28

### DISCUSSION THREAD

**Discussion by JayDoubleu** [okd-project/okd#1006] [Q&A]
When trying to install single node OKD with Assisted Installer it returns below error:

`Machine network CIDR xxx.xxx.xxx.xxx/29: Address mask size must be between 1 to 28 and must include at least 16 addresses`
This error only seem to be appearing in version `4.8.17` while on `4.9.4` only returns non descriptive `machine network CIDR : invalid CIDR address`. I'm also unable to find documentation describing this requirement.

Is /28 definitely required when on single node cluster ?

**Comment by vrutkovs** [comment]
I don't think Assisted Installer supports OKD (yet), at least we don't have `4.8.17` or `4.9.4` versions

**Reply by JayDoubleu** [reply]
Well google for "OKD single node" sent me to "https://docs.okd.io/latest/installing/installing_sno/install-sno-installing-sno.html"

**Reply by JayDoubleu** [reply]
I already installed it via assistant installer but on a larger subnet and seemed to work fine but doesnt seem to install OKD but trial version of OC.

I take it that the https://docs.okd.io/ is a copy of openshift docs meant to be modified for OKD ?

**Comment by tunicashashi** [comment]
Is there any plan to bring assisted installer for OKD as well in coming releases?
